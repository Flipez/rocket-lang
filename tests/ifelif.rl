i = 0
while (i<2)
  if (i==0)
    puts("if/elif: if block")
  elif (i==1)
    puts("if/elif: elif block")
  end

  i = i+1
end
