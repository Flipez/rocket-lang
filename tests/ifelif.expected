"if/elif: if block"
"if/elif: elif block"
nil
